src/styles/blocks/page.scss

```diff
@@ -0,0 +1,81 @@
+.page {
+  font-family: 'Inter', sans-serif;
+  scroll-behavior: smooth;
+
+  &__body {
+    margin: 0;
+
+    &--with-menu {
+      overflow: hidden;
+    }
+  }
+
+  &__menu {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+
+    transform: translateX(+100%);
+    transition: transform 0.3s;
+
+    @include onDesctop {
+      transform: translateX(-100%);
```

Comment:
why do you hide it on different sides for different widths? It seems like it leads to 3rd issue [here](https://github.com/mate-academy/layout_miami/pull/2477#pullrequestreview-1426460831)